easypost/__init__.py

```diff
@@ -272,8 +273,10 @@ def request_raw(self, method, url, params=None, apiKeyRequired=True):
                 'at contact@easypost.com for assistance.')
 
         abs_url = self.api_url(url)
+        if (url == "/rating/v1/rates"):
```

Comment:
I agree, it's messy as it is now. Also, comparing/using random strings in line is 99.9% of the time bad form. I would convert them to constants.